consensus/wal.go

```diff
@@ -17,6 +17,10 @@ import (
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
+const (
+	maxMsgSizeBytes = 1024 // 1MB, must be greater than BlockPartSizeBytes
```

Comment:
Um, that is 1KiB. 1MiB = 1024 * 1024

Reply:
Yeh let's just put 1MiB and a note about how to find the largest expected size (ie. `params.BlockGossip.BlockPartSizeBytes` + a few bytes). And maybe a corresponding comment or a panic/sanity check on this somewhere?